libs/static/src/chains.ts

```diff
@@ -109,7 +109,7 @@ export const existentialDeposit: Record<Prefix, number> = {
   rmrk: 333333333,
   ahk: 333333333,
   dot: 10000000000,
-  ahp: 1000000000,
+  ahp: 100000000,
```

Comment:
Can we use a numeric separator here? `100_000_000`

Reply:
```suggestion
  ahp: 1e8,
```

Reply:
updated 85e5b10